testsuite/gnat2goto/README.md

```diff
@@ -89,6 +89,24 @@ Requirements
 
 Running tests
 -------------
+
+Ensure that both gnat2goto and cbmc are on the path:
+
+```
+export PATH=${PATH}:/path/to/gnat2goto/lib/cbmc/build/bin:/path/to/gnat2goto/gnat2goto/install
+```
+
+From within [gnat2goto/testsuite/gnat2goto](.) run:
+
+```
+$ ./testsuite.py -j 4
+```
+where ```-j``` sets the number of tests to run in parallel (default is 1).
+
+This will output which tests pass and fail. If a test fails, you can inspect the output 
+inside `out`. Each test will produce a file called `test_name.diff` which contains a git
+diff between the expected an actual output. 
```

Comment:
You can also add other options to `testsuite.py` such as `--timeout 60 --diffs --enable-color` (taken from CI) - which means "timeout any single test case that takes longer than 60 seconds, output diffs for any XFAIL, FAIL, or UPASS result, and use colour in the output"

Reply:
Personally find the `--diffs` a bit overwhelming for a first run, perhaps I could link here to fuller docs about the testuite if such a file exists?

Reply:
Yeah, it's a bit overly verbose :-) Happy either way, I was mostly just checking you knew about those options.

Reply:
Couldn't find any so mentioned it as an alternative to inspecting diffs. Personally I'd have `--diffs` only show difference in FAIL and UPASS (since XFAIL diffs are constant).